tests/nodeos_under_min_avail_ram.py

```diff
@@ -167,8 +169,15 @@ def setName(self, num):
             try:
                 trans=nodes[0].pushMessage(contract, action, data, opts)
                 if trans is None or not trans[0]:
+                    timeOutCount+=1
+                    if timeOutCount>=3:
+                       Print("Failed to push create action to eosio contract for %d consecutive times, looks like nodeos already exited." % (timeOutCount))
```

Comment:
This should be a errorExit(...) call so that we report that we are failing because we failed pushing the message <timeOutCount> times.

Reply:
In my understanding this test script does expect nodeos exits due to no available memory, then the script will continue to check how many rounds of transactions have been pushed before nodeos exits. am I correct?

Reply:
Yes, so  if it gets a None transaction, nodeos hasn't shutdown, so it is something else going wrong, and we don't want to evaluate anything more. and should stop.

Reply:
Ok, just ran this and not sure why the failure is occurring different than when I first wrote it, but @taokayan has it right.